rdt/transformers/null.py

```diff
@@ -47,14 +47,16 @@ def fit(self, data):
             data (pandas.Series or numpy.ndarray):
                 Data to transform.
         """
+        null_values = data.isnull().values
+        self.nulls = null_values.any()
+        contains_not_null = (~null_values).any()
```

Comment:
I'm curious about whether `not null_values.all()` makes any difference.

Reply:
I'll try it out

Reply:
looks _slightly_ faster. I can change that